rllib/models/catalog.py

```diff
@@ -181,6 +181,13 @@
     # backward compatibility to old configs. This yields different models than past
     # versions of RLlib.
     "encoder_latent_dim": None,
+    # Whether to always check the inputs and outputs of RLlib's default models for
```

Comment:
Wait, this is NOT just for the default models, correct?

Reply:
always_check_shapes only triggers spec checking for the models right now.
But we can make the switch system-wide. I think it's a good idea since we only need it for debugging anyway. 
Wdyt?

Reply:
Ok, understood. Thanks for clarifying. Ok with me to leave as-is for now and make this system wide once we move away from the old config dict.